code/API_definitions/device-status.yaml

```
@@ -543,170 +630,182 @@ components:
               format: date-time
               description: date time when subscription will expire or expired
           required:
-            - eventSubscriptionId
+            - subscriptionId
             - type
 
-    EventSubscriptionAsync:
+    SubscriptionAsync:
       description: Response for a device status operation managed asynchronously (Creation or Deletion)
       type: object
       properties:
-          eventSubscriptionId:
-              $ref: '#/components/schemas/EventSubscriptionId'
+        subscriptionId:
+          $ref: "#/components/schemas/SubscriptionId"
 
-    
-    EventSubscriptionId:
+    SubscriptionId:
       type: string
       description: The event subscription identifier.
-    
-    EventTime:
-      format: date-time
-      type: string
-      description: The time when the event was notified.
-    
-    Notification:
-      description: The notification callback.
-      type: object
-      required:
-        - event
-      properties:
-        eventSubscriptionId:
-          $ref: '#/components/schemas/EventSubscriptionId'
-        event:
-          $ref: '#/components/schemas/Event'
+      example: qs15-h556-rt89-1298
 
-    Event:
-      description: Generic event structure
+    CloudEvent:
+      description: The notification callback
       required:
-        - eventType
-        - eventTime
-        - eventDetail
+        - id
+        - source
+        - specversion
+        - type
+        - time
       properties:
-        eventType:
+        id:
           type: string
-        eventTime:
-          $ref: '#/components/schemas/EventTime'
-        eventId:
+          description: identifier of this event, that must be unique in the source context.
+        source:
+          $ref: "#/components/schemas/Source"
+        type:
           type: string
-          format: uuid
-        eventDetail:
+          description: "type of event as defined in each CAMARA API"
+          example: org.camaraproject.device-status.v0.roaming-status
+        specversion:
+          type: string
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          example: 1.0
```

string examples are sometimes in quotes sometimes not, not consistent (this is not new with this PR, just noticed :-) )

---

Without quotes it is rendered by Swager as integer, just: 1

```suggestion
          example: '1.0'
```